gateway/1.6/modules/ROOT/pages/policies-included-external-authorization.adoc

```diff
@@ -0,0 +1,124 @@
+= External Authorization Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+:keywords: external authorization, authorization, flex gateway, gateway, policy
+
+[width="100%", cols="5,15"]
+|===
+>s|Policy name | External Authorization
+>s|Summary | Authenticates requests with an external gRPC or HTTP authorization service
+>s|Category | Security
+>s| First Flex Gateway version available | v1.6.0
+>s| Returned Status Codes | 403 - Forbidden, invalid client application credentials
+|===
+
+== Summary
+
+The External Authorization policy sends incoming client HTTP requests to an external gRPC of HTTP authorization service to determine if the client HTTP request is authorized to access the upstream service. For authorized accesses, Flex Gateway returns the response from the upstream service. For unauthorized accesses, Flex Gateway returns a `403` forbidden request error. 
+
+If it is necessary to send request headers to the authorization service, such as an authorization token header, you can configure what headers from the client HTTP requests are sent to the authorization service by using the *Allowed Headers* parameter.
+
+The External Authorization policy can also modify the client HTTP requests header with additional information returned from the authorization service. If the authorization service returns headers with the authorization response, Flex Gateway either appends these headers to the client HTTP request or overwrites the header if it is already present in the HTTP request. For example, if an authorized user sends a request with an access token corresponding to a username, the authorization service can return a `username` header for Flex Gateway to append to the client HTTP request.
+
+The External Authorization policy is an extension of the External Authorization filter native to Envoy. To learn more about the External Authorization filter, see https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/ext_authz_filter[External Authorization Filter^]. Not all External Authorization filter parameters are available for the External Authorization policy. To find what parameters are configurable, see <<configuring-policy-parameters, Configuring Policy Parameters>>. 
+
+NOTE: If other policies modify the incoming HTTP request, ensure you order the policies depending on if these modification should be present when the request is sent to the authorization service. For more information about ordering policies, see xref:policies-reorder.adoc[].
```

Comment:
```suggestion
NOTE: If other policies modify the incoming HTTP request, ensure that you order the policies depending on if these modification should be present when the request is sent to the authorization service. For more information about ordering policies, see xref:policies-reorder.adoc[].
```

Reply:
Can you reword the paragraph above? It is not clear how the policies should be ordered. Do you put the policies with modifications first?